src/mmds/Cargo.toml

```diff
@@ -14,6 +14,7 @@ serde = { version = ">=1.0.27", features = ["derive"] }
 serde_json = ">=1.0.9"
 versionize = ">=0.1.6"
 versionize_derive = ">=0.1.3"
+zeroize = "1.4.3"
```

Comment:
I don't want use to take this dependency on when we can implement the needed functionality with very little cost.

Reply:
How about we decuple the zeroization commit from this PR in order to not block the other fixes from being merged? I will create a separate PR to clear key's contents if applicable.